src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -810,6 +831,17 @@ public Descriptor getDefaultPodRetention() {
             return jenkins.getDescriptor(PodRetention.getPodTemplateDefault().getClass());
         }
 
+        @SuppressWarnings("unused") // Used by jelly
+        @Restricted(DoNotUse.class) // Used by jelly
+        public Descriptor getDefaultYamlMergeStrategy() {
+            Jenkins jenkins = Jenkins.getInstanceOrNull();
+            if (jenkins == null) {
+                return null;
+            }
+            return jenkins.getDescriptor(YamlMergeStrategy.defaultStrategy().getClass());
```

Comment:
Return the `Describable`, not the `Descriptor`.

> If specified, this will be chosen as the default value in case the current selection is null. The default can be a specific instance or a descriptor e.g. ${descriptor.defaultSettingsProvider} or ${descriptor.defaultSettingsProvider.descriptor}. In the later case, the from input fields will be empty.